hoprd/rest-api/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "hoprd-api"
-version = "4.0.0"
+version = "4.1.0"
```

Comment:
This is a patch change, it adds stuff, does not remove anything and is backwards compatible.